packages/web3-providers/types/index.d.ts

```diff
@@ -55,6 +55,7 @@ export class SocketProviderAdapter extends AbstractProviderAdapter {
 }
 
 export class BatchRequest {
+    constructor();
     constructor(provider: provider, jsonRpcMapper: JsonRpcMapper, jsonRpcResponseValidator: JsonRpcResponseValidator);
```

Comment:
That's not the current constructor but I will change it by my self after I wrote all the providers tests.

Reply:
that constructor was what it as before, yeah if you can change it what would be great 👍